recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@ package:
   version: 3.4.2
 
 build:
-  number: 0
+  number: 1
```

Comment:
So we don't do build number bumps with this package only version changes. That is because this feedstock is its source. Instead of doing this we should change the version to say 3.5.0.